generator/gm/gm_global_a_powertrain.dbc

```diff
@@ -64,7 +64,7 @@ BO_ 189 EBCMRegenPaddle: 7 K17_EBCM
  SG_ RegenPaddle : 7|4@0+ (1,0) [0|0] ""  NEO
 
 BO_ 190 ECMAcceleratorPos: 6 K20_ECM
- SG_ BrakePedalPos : 15|8@0+ (1,0) [0|0] "sticky"  NEO
+ SG_ BrakePedalPos : 15|8@0+ (1,0) [0|0] ">=8 disengages ACC"  NEO
```

Comment:
this is the unit field, move to comment (CM_ line)

Reply:
Whoops, fixed, squashed